### ISSUE TITLE
How to use toner with html recipe

### ISSUE THREAD

**Issue by dlong500** [jsreport/toner#4]
I don't see any documentation for how to use toner with the html recipe.  If I try the following (based on the wkhtmltopdf example) the `res` variable is undefined and `err` outputs `[Error: Recipe 'html' not found]`:

``` javascript
    var toner = require("toner")();
    toner.engine("jsrender", require("toner-jsrender"));

    toner.render({
        template: { 
            engine: "jsrender",
            recipe: "html", 
            content: "<h1>{{:foo}}</h1>"
        },
        data: { foo: "hello world"}
    }, function(err, res) {
        console.log(res);
        console.log(err);
    });
```

I don't see anything in npm for installing an html recipe.

My goal is to be able to use the same template to generate either html or pdf content in a node app (specifically using hapi.js).

**Comment by pofider** [comment]
I have updated documentation, thanks.

To use html you can use following without any additional npm package installation:

``` js
toner.recipe("html", toner.htmlRecipe);
toner.render({
  template: {
    recipe: 'html'   
    ...
  }
```

**Comment by dlong500** [comment]
Sorry, I don't know if I'm still missing something.

I've added the `toner.recipe("html", toner.htmlRecipe);` line and I'm still getting `[Error: Recipe 'html' not found]`

**Comment by pofider** [comment]
Ups. My fault! Right now you need to do it in the following way:

``` js
toner.recipe("html", require("toner").htmlRecipe);
```

I will add the `htmlRecipe` property to toner instance as well, please keep this bug open.

**Comment by dlong500** [comment]
Ok, great.

I suppose the same might apply for the "none" engine?

So for now it should be:
`toner.engine("none", require("toner").noneEngine);`
instead of
`toner.engine("none", toner.noneEngine);`

Is that correct?

**Comment by pofider** [comment]
Yes exactly

**Comment by pofider** [comment]
Released in toner 0.1.3
